virtualcluster/pkg/syncer/apis/config/types.go

```diff
@@ -70,6 +70,9 @@ type SyncerConfiguration struct {
 
 	// Super cluster rest config
 	RestConfig *rest.Config
+
+	// The maximum length of time to wait before giving up on a server request. A value of "" means use default.
```

Comment:
Can you explain the unit of timeout in the comment (second or millisecond, etc.)?

Reply:
Yes, I can follow up with something, the user can supply anything from `5s`, `5m`, `1h`, anything that `time.ParseDuration` accepts.